internal/wasm/func_validation.go

```diff
@@ -759,7 +759,7 @@ func (m *Module) validateFunctionWithMaxStackValues(
 				var params []ValueType
 				switch miscOpcode {
 				case OpcodeMiscMemoryInit, OpcodeMiscMemoryCopy, OpcodeMiscMemoryFill, OpcodeMiscDataDrop:
```

Comment:
quick comment above here and elsewhere which feature/release these are about as it is confusing since Wasm mixes things so much.

Ex. // Handle opcodes added in bulk-memory-operations/WebAssembly 2.0